pyliquibase/__init__.py

```diff
@@ -96,8 +88,7 @@ def _cli(self):
             self.liquibase_dir + "/liquibase.jar",
             self.liquibase_lib_dir + "/*",
             self.liquibase_internal_dir + "/*",
-            self.liquibase_internal_lib_dir + "/*",
-            self.jdbc_drivers_dir + "/*"]
```

Comment:
we should add it to the list if self.jdbc_drivers_dir is not None, provided by user.